package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@bitfinex/lib-util-err-js",
+  "version": "0.0.1",
+  "description": "Bitfinex lib utility for handling error messages",
+  "main": "index.js",
+  "scripts": {
+    "docs": "jsdoc2md --no-cache --files ./src/*.js ./src/**/*.js > docs/DEFINITIONS.md",
```

Comment:
maybe its better to not check those generated files in? they clutter PRs and usually docs are generated locally and the compiled result is not checked into source control

Reply:
cris started with checking in those docs in in the HF and its a nightmare, especially when the docs are large and someone runs it and the `generated at Friday` comment is everything that has changed

Reply:
Sure I'll add docs dir to gitignore then